### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by zainabbehzad**
--Tribute page project
** In this project first of all I have added a repository by the name of Tribute-page.
**Then I have added the linters configurations.
**Created an HTML file (index.html) added all html tags in this file and it was information about Dr. Norman Borlaug.
**Then created CSS file ( styles.css) and added all styles by this in my project.

### CODE REVIEW

File: index.html
Comment (ahzamir) [inline index.html:1]: - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @zainabbehzad  🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Copy your HTML and CSS file code and paste it to the freecodecamp website Tribute Page project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Tribute Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
❌ Your code is clean
**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by shogof** [comment]
Hi Zainab Behzad,

I've reviewed your Tribute Page project on GitHub, and overall, you've done a commendable job 👏🏻🦾. Here's a breakdown of my feedback:

Positives:
Great Design: Your Tribute Page has a clean and professional design. The layout is well-organized, and the use of images enhances the visual appeal.
Meaningful Repo: You've created a repository with an appropriate name for your project, which makes it easy to identify and access.
Gitflow: It's good to see that you've followed correct Gitflow practices, creating a development branch and making it the default branch. Also, creating and committing your work on a feature branch is a good practice.
Linter Configuration: You've successfully set up linters and fixed any errors, ensuring code quality and consistency.
PR Description: Your pull request description is informative, providing details about the changes made and features added.

Areas of Improvement:

Tribute Page Requirements: Although you've implemented a Tribute Page and covered significant aspects of Dr. Norman Borlaug's life, ensure that you've met all the specified requirements. Double-check against the project guidelines to ensure nothing is missed.
Code Cleanliness: While your HTML and CSS codes are functional, consider optimizing them further for readability and maintainability. Organize your code into logical sections and consider using comments to explain complex or critical parts.
Bugs or Issues:

No specific bugs or issues were mentioned in your provided information.
Overall, you've demonstrated a good understanding of HTML and CSS concepts, and your project shows promise. Keep refining your skills and paying attention to project requirements and code quality.

Feel free to reach out if you have any questions or need further clarification on any aspect of the feedback.

Best regards,
Shogofa Muradi

File: index.html
Comment (zainabbehzad) [reply index.html:1]: @ahzamir @ahzia 
please review my project.
My CSS file name is styles.css as in freecodecamp this file has the same name, if I set its name to style.css I should change its name in the link from style.css to styles.css.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@zainabbehzad 

- Althought you have solved the requirement issue and now your code passes all checks on freecodecamp website but still your code is not clean:
- And for cleaning you code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!
  
  - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Comment by zainabbehzad** [comment]
Hello sir
Please review my project.

**Review by ahzamir — APPROVED** [review]
### Hi @zainabbehzad 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**